lib/Debugger.ProtocolHandler/RuntimeImpl.cpp

```diff
@@ -315,6 +316,38 @@ namespace JsDebug
         return true;
     }
 
+    JsValueRef JsJSONStringfyObject(JsValueRef value, JsValueRef* error)
```

Comment:
The name is misspelled. It's also inconsistent with regard to case. Might I suggest `JsObjectToJson()` which is much more readable.